Package breaks apps deployed to Cloudflare Pages

Hi, @ssssota 

Great work on `svelte-exmarkdown`. Thank you.

As I was trying to use it for my use case, I ran into some challenges, some highlighted in #113, but so far, the most pressing is the fact that when using this package in the Cloudflare Pages environment, they break, for reasons, more or less, related to how CF doesn't support a true node runtime - or so I hear.

These are the errors output in the console.

```bash
20:14:27.378 | ✘ [ERROR] Could not resolve "node:process"
-- | --
20:14:27.379 |  
20:14:27.379 | .svelte-kit/output/server/entries/pages/_page.svelte.js:3:17:
20:14:27.380 | 3 │ import proc from "node:process";
20:14:27.380 | ╵                  ~~~~~~~~~~~~~~
20:14:27.381 |  
20:14:27.381 | The package "node:process" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
20:14:27.381 |  
20:14:27.381 | ✘ [ERROR] Could not resolve "node:path"
20:14:27.381 |  
20:14:27.382 | .svelte-kit/output/server/entries/pages/_page.svelte.js:4:17:
20:14:27.382 | 4 │ import path from "node:path";
20:14:27.382 | ╵                  ~~~~~~~~~~~
20:14:27.382 |  
20:14:27.382 | The package "node:path" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
20:14:27.382 |  
20:14:27.382 | ✘ [ERROR] Could not resolve "node:url"
20:14:27.383 |  
20:14:27.383 | .svelte-kit/output/server/entries/pages/_page.svelte.js:5:30:
20:14:27.383 | 5 │ import { fileURLToPath } from "node:url";
20:14:27.383 | ╵                               ~~~~~~~~~~
20:14:27.383 |  
20:14:27.383 | The package "node:url" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
20:14:27.383 |  
20:14:27.384 | error during build:
20:14:27.384 | Error: Build failed with 3 errors:
20:14:27.384 | .svelte-kit/output/server/entries/pages/_page.svelte.js:3:17: ERROR: Could not resolve "node:process"
20:14:27.384 | .svelte-kit/output/server/entries/pages/_page.svelte.js:4:17: ERROR: Could not resolve "node:path"
20:14:27.385 | .svelte-kit/output/server/entries/pages/_page.svelte.js:5:30: ERROR: Could not resolve "node:url"
```

Are you aware of a project that used this package running on Cloudflare?

If not, is there a fix for people who use Cloudflare for hosting/deployments?

Thank you for the consideration.

--- a/.changeset/healthy-cats-relax.md
+++ b/.changeset/healthy-cats-relax.md
@@ -1,0 +1,5 @@
+---
+"svelte-exmarkdown": patch
+---
+
+build: prebundle external package

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
 	"scripts": {
 		"dev": "vite dev",
 		"build": "vite build",
-		"package": "svelte-package",
+		"package": "svelte-package && node scripts/prebundle.js",
 		"preview": "vite preview",
 		"prepare": "svelte-kit sync && husky install",
 		"clean": "rimraf build coverage dist",
@@ -46,6 +46,7 @@
 		"@typescript-eslint/parser": "^6.4.1",
 		"@vitest/coverage-v8": "^0.34.3",
 		"@vitest/ui": "^0.34.3",
+		"esbuild": "^0.19.10",
 		"eslint": "^8.47.0",
 		"eslint-config-prettier": "^9.0.0",
 		"eslint-plugin-svelte": "^2.33.0",

--- a/scripts/prebundle.js
+++ b/scripts/prebundle.js
@@ -1,3 +1,4 @@
+// @ts-check
 import { build } from 'esbuild';
 
 const prebundleTargets = ['dist/utils.js', 'dist/gfm/index.js'];
@@ -10,7 +11,8 @@ Promise.all(
 			bundle: true,
 			outfile: target,
 			allowOverwrite: true,
-			format: 'esm'
+			format: 'esm',
+			platform: 'neutral'
 		});
 	})
 );

--- a/scripts/prebundle.js
+++ b/scripts/prebundle.js
@@ -1,0 +1,16 @@
+import { build } from 'esbuild';
+
+const prebundleTargets = ['dist/utils.js', 'dist/gfm/index.js'];
+
+Promise.all(
+	prebundleTargets.map((target) => {
+		console.log(`Prebundling ${target}`);
+		return build({
+			entryPoints: [target],
+			bundle: true,
+			outfile: target,
+			allowOverwrite: true,
+			format: 'esm'
+		});
+	})
+);

